.github/CODEOWNERS

```diff
@@ -3,14 +3,14 @@
 /scripts/ @Azure/bicep-admins @Azure/avm-core-team-technical-bicep
 /avm/ @Azure/avm-core-team-technical-bicep
 /avm/utilities/ @Azure/avm-core-team-technical-bicep
+/avm/ptn/authorization/policy-assignment/ @Azure/avm-ptn-authorization-policyassignment-module-owners-bicep @Azure/avm-core-team-technical-bicep
```

Comment:
Soft-block until team is created and placed in hierachy

Reply:
This team has been created.

Reply:
Including the request to add it to the correct parent team?

Reply:
My understanding is that Mate gets automated notifications when we create the teams.

Reply:
He does when you try to add it to the hierarchy, yes. @matebarabas, please take a look once you have a moment :)

Reply:
I can confirm that both the -owners- and -contributors- teams have been created and assigned to their respective parents.

Reply:
And yet the file does show errors. @arnoldna can you double check that the names match? Maybe there is a typo somewhere

Reply:
It seems to match what is in here. https://azure.github.io/Azure-Verified-Modules/indexes/bicep/bicep-pattern-modules/#for-module-owners--contributors

Reply:
I ended up creating a new team to match the code owners file. This should not be a conflict any longer